boards/qemu/qemu_x86/board.yml

```diff
@@ -23,8 +23,5 @@ boards:
   - name: qemu_x86_tiny
     socs:
       - name: ia32
-    revision:
-      format: custom
-      default: ''
-      revisions:
```

Comment:
These changes can be merged into the main commit

Reply:
> These changes can be merged into the main commit

? this piece of code is already outdated

Reply:
imo its better to keep this commit separate to track the board naming change
(`qemu_x86_tiny@768` board revision to `qemu_x86_tiny/ia32/768`)

Reply:
For future going forward it should just be part of the original commit, these commits will only exist in the collab-hwm branch, when it is ready for merging into `main` every commit is going to be squashed into one and doing it like above just causes confusion for the reviewer

Reply:
> For future going forward it should just be part of the original commit, these commits will only exist in the collab-hwm branch, when it is ready for merging into `main` every commit is going to be squashed into one and doing it like above just causes confusion for the reviewer

quemux86_tiny@768 rename caused by HWMv2 migration is just one of 6 commits here which all be squashed later anyway. What other commit it have to be a part of now ?